docs/deployment/primary-branch.md

```diff
@@ -0,0 +1,36 @@
+---
+slug: primary-branch
+title: Set a primary branch
+hide_title: true
+description: Set your primary or default branch to ensure Semgrep full scans display accurate counts and deduplicated findings.
+tags:
+  - Core deployment
+---
+
+# Set a primary branch
+
+A **primary branch** is the base or target branch for pull and merge requests. It is usually referred to as a **default branch** or **trunk** by your source code manager (SCM). Typical names for a primary branch include `dev`, `production`, or `develop`.
+
+In many cases, Semgrep automatically detects primary branches. If you have projects (repositories) with unique primary branch names, you can set them through the Semgrep web app.
+
+:::note
+- This feature is in **private beta**. To request access, contact your Technical Account Manager or your Account Executive and let them know you'd like to join the primary branch beta.
+- Primary branches are set on a **per-project** basis. They cannot be set in bulk.
```

Comment:
Not sure what the best practice here is, but I wonder if it's worth calling out that they can also do this via the public API?

Reply:
Hi @pangngan I was looking in https://semgrep.dev/api/v1/docs/ for the endpoint, but there were no relevant results for `default` or `primary` - any pointers here? A link would be helpful for the readers. 🙏

Reply:
Yep, they can also set this via the PATCH endpoint here: https://semgrep.dev/api/v1/docs/#tag/Project/operation/semgrep_app.saas.handlers.repository.openapi_patch_project

Reply:
Thank you!